cmd/main.go

```diff
@@ -95,6 +95,12 @@ func main() {
 					Value:       1000000, // nolint:gomnd
 					DefaultText: "1000000",
 				},
+				&cli.Uint64Flag{
+					Name:        "writetout",
+					Usage:       "timeout for write operations on client connections in ms (0=no timeout)",
```

Comment:
This may be a "duration" instead of a Uint64. It that way you can specify the units and avoid multiplying * time.Millisecond later.

Reply:
This is just a DS CLI testing tool, this new parameter behaves like another one existing one.